### PR TITLE
Makes credentials available in the deployed game

### PR DESCRIPTION
**Pull request by jog1t**
Closes GDT-55

### CODE REVIEW

**Review by AngelOnFira — APPROVED** [review]
[firsttry](https://app.graphite.dev/macros/rivet-gg)

File: addons/rivet/devtools/rivet_plugin_bridge.gd [hunk 1]
```diff
@@ -7,10 +7,16 @@
 
 signal bootstrapped
 
+const RIVET_CONFIGURATION_PATH: String = "res://.rivet"
+const RIVET_CONFIGURATION_FILE_PATH: String = "res://.rivet/config.gd"
+const RIVET_DEPLOYED_CONFIGURATION_FILE_PATH: String = "res://.rivet_config.gd"
+const SCRIPT_TEMPLATE: String ="extends RefCounted\nconst api_endpoint: String = \"{api_endpoint}\"\nconst namespace_token: String = \"{namespace_token}\"\nconst cloud_token: String = \"{cloud_token}\"\nconst game_id: String = \"{game_id}\""
```
Comment (NathanFlurry) [inline addons/rivet/devtools/rivet_plugin_bridge.gd:13]: Multiline string

File: addons/rivet/devtools/rivet_plugin_bridge.gd [hunk 2]
```diff
@@ -7,10 +7,16 @@
...
 const _global := preload("../rivet_global.gd")
 
 static var instance = RivetPluginBridge.new()
 
+static var cli
```
Comment (NathanFlurry) [inline addons/rivet/devtools/rivet_plugin_bridge.gd:18]: Delete me!

File: addons/rivet/rivet_global.gd [hunk 3]
```diff
@@ -7,15 +7,15 @@ extends Node
...
-
-var cli := _cli.new()
 
 var cloud_token: String
 var namespace_token: String 
 var game_id: String
 var api_endpoint: String
 
+# This variable is only accessible from editor's scripts, please do not use it in your game.
+var cli
```
Comment (NathanFlurry) [inline addons/rivet/rivet_global.gd:17]: Would it make sense to name this `_cli` or `_internal_cli` or something similar?

Can this have a type?

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

**Review by NathanFlurry — APPROVED** [review]

**Pull request merged**